recipe-server/client/control_new/components/data/QueryRecipes.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { isEqual } from 'underscore';
+
+import { fetchFilteredRecipesPage } from 'control_new/state/recipes/actions';
+
+
+class QueryRecipes extends React.Component {
+  static propTypes = {
+    fetchFilteredRecipesPage: PropTypes.func,
+    filters: PropTypes.object,
+    pageNumber: PropTypes.number,
+  }
+
+  componentWillMount() {
+    const { filters, pageNumber } = this.props;
+    this.props.fetchFilteredRecipesPage(pageNumber, filters);
```

Comment:
there's a conflict with the action creator existing in a higher scope (due to being imported with the same name). i'm open to suggestions to avoid this because this is annoying.